Tests/App/AppKernel.php

```diff
@@ -25,6 +25,7 @@ public function registerBundles(): array
             new \Symfony\Bundle\SecurityBundle\SecurityBundle(),
             new \Symfony\Bundle\MonologBundle\MonologBundle(),
             new \Symfony\Bundle\TwigBundle\TwigBundle(),
+            /* @phpstan-ignore-next-line */
             new \FOS\UserBundle\FOSUserBundle(),
```

Comment:
This could be better I guess to do:
```suggestion
            if (class_exists(\FOS\UserBundle\FOSUserBundle::class)) {
                new \FOS\UserBundle\FOSUserBundle(),
            }
```

Reply:
yes makes more sense :+1: